tests/test_api.py

```diff
@@ -85,6 +85,7 @@ def test_status_format(self, client, endpoint, method):
         ('/biospecimens', ['participant', 'sequencing_center']),
         ('/sequencing-experiments', ['sequencing_center']),
         ('/genomic-files', ['biospecimen', 'sequencing_experiment']),
```

Comment:
Read group link is formatted like a parent link so should be added to the parent link tests for genomic file: 
```
('/genomic-files', ['biospecimen', 'sequencing_experiment', 'read_group']),
```